src/shadowbox/server/server_access_key.ts

```diff
@@ -70,14 +67,15 @@ function makeAccessKey(hostname: string, accessKeyJson: AccessKeyJson): AccessKe
       accessKeyJson.id, accessKeyJson.name, accessKeyJson.metricsId, proxyParams);
 }
 
-function makeAccessKeyJson(accessKey: AccessKey): AccessKeyJson {
+function accessKeySerializedJson(accessKey: AccessKey): AccessKeyJson {
```

Comment:
nit: I think the previous name was clearer since it denotes an action

Reply:
Done in  c9ae52d